shenyu-examples/shenyu-examples-dubbo/shenyu-examples-apache-dubbo-service-client/src/main/resources/application.yml

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+server:
+  port: 8011
+  address: 0.0.0.0
+  servlet:
+    context-path: /
+spring:
+  main:
+    allow-bean-definition-overriding: true
+dubbo:
+  registry:
+    address: zookeeper://localhost:2181
+    port: 20888
+    
+shenyu:
```

Comment:
remove this， it springboot starter

Reply:
use spring boot to write this example and place all argument in application.yml, and this will be read in spring-dubbo.xml dubbo:register and dubbo-protocol

Reply:
remove shenyu-client config